snmp/datadog_checks/snmp/data/default_profiles/apc-pdu.yaml

```diff
@@ -0,0 +1,227 @@
+        tag: r_pdu_status_outlet_state
+        mapping:
+          1: outlet_load_normal
+          2: outlet_load_low
+          3: outlet_load_near_overload
+          4: outlet_load_overload
+  - MIB: PowerNet-MIB
+    table:
+      OID: 1.3.6.1.4.1.318.1.1.26.10.2.2
+      name: rPDU2SensorTempHumidityStatusTable
+    symbols:
+      - OID: 1.3.6.1.4.1.318.1.1.26.10.2.2.1.7
+        name: rPDU2SensorTempHumidityStatusTempF
+      - OID: 1.3.6.1.4.1.318.1.1.26.10.2.2.1.8
+        name: rPDU2SensorTempHumidityStatusTempC
+      - OID: 1.3.6.1.4.1.318.1.1.26.10.2.2.1.10
+        name: rPDU2SensorTempHumidityStatusRelativeHumidity
+    metric_tags:
+      - tag: r_pdu2_sensor_temp_humidity_status_name
+        column:
+          OID: 1.3.6.1.4.1.318.1.1.26.10.2.2.1.3
+          name: rPDU2SensorTempHumidityStatusName
+      - tag: r_pdu2_sensor_temp_humidity_status_type
+        column:
+          OID: 1.3.6.1.4.1.318.1.1.26.10.2.2.1.5
+          name: rPDU2SensorTempHumidityStatusType
+        mapping:
+          1: temperature_only
+          2: temperature_humidity
+          3: comms_lost
+          4: not_installed
+      - column:
+          OID: 1.3.6.1.4.1.318.1.1.26.10.2.2.1.9
+          name: rPDU2SensorTempHumidityStatusTempStatus
+        tag: r_pdu2_sensor_temp_humidity_status_temp_status
+        mapping:
+          1: not_present
+          2: below_min
+          3: below_low
+          4: normal
+          5: above_high
+          6: above_max
+      - column:
+          OID: 1.3.6.1.4.1.318.1.1.26.10.2.2.1.11
+          name: rPDU2SensorTempHumidityStatusHumidityStatus
+        tag: r_pdu2_sensor_temp_humidity_status_humidity_status
+        mapping:
+          1: not_present
+          2: below_min
+          3: below_low
+          4: normal
+          5: above_high
+          6: above_max
+
+metric_tags:
+  - tag: r_pdu_ident_name
+    OID: 1.3.6.1.4.1.318.1.1.12.1.1.0
+    symbol: rPDUIdentName
+  - tag: r_pdu_ident_hardware_rev
+    OID: 1.3.6.1.4.1.318.1.1.12.1.2.0
+    symbol: rPDUIdentHardwareRev
+  - tag: r_pdu_ident_firmware_rev
+    OID: 1.3.6.1.4.1.318.1.1.12.1.3.0
+    symbol: rPDUIdentFirmwareRev
+  - tag: r_pdu_ident_model_number
+    OID: 1.3.6.1.4.1.318.1.1.12.1.5.0
+    symbol: rPDUIdentModelNumber
+  - tag: r_pdu_ident_serial_number
+    OID: 1.3.6.1.4.1.318.1.1.12.1.6.0
+    symbol: rPDUIdentSerialNumber
+  - tag: r_pdu_ident_device_rating
+    OID: 1.3.6.1.4.1.318.1.1.12.1.7.0
+    symbol: rPDUIdentDeviceRating
+  - tag: r_pdu_ident_device_num_outlets
+    OID: 1.3.6.1.4.1.318.1.1.12.1.8.0
+    symbol: rPDUIdentDeviceNumOutlets
+  - tag: r_pdu_ident_device_num_phases
+    OID: 1.3.6.1.4.1.318.1.1.12.1.9.0
+    symbol: rPDUIdentDeviceNumPhases
+  - tag: r_pdu_ident_device_num_breakers
+    OID: 1.3.6.1.4.1.318.1.1.12.1.10.0
+    symbol: rPDUIdentDeviceNumBreakers
+  - tag: r_pdu_ident_device_breaker_rating
+    OID: 1.3.6.1.4.1.318.1.1.12.1.11.0
+    symbol: rPDUIdentDeviceBreakerRating
```

Comment:
Q: Did you checked that all those tags have values that are stable?
We should be careful since if the values have high cardinality, it might lead to high metric contexts.

Reply:
Should be good, but removed some to be safe. We can add them back later if needed.